account_payment_order/views/account_payment_line.xml

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
+
```

Comment:
Don't add useless empty lines for nothing. In fact, auto-formatters tend to remove them (not for now on OCA), but in this case that they don't exist before, it's not correct to add them.